scripts/check_change_log/main.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+import glob
+import os
+import subprocess as sp
+import time
+from pypi_tools.pypi import PyPIClient
+from pathlib import Path
+
+
+def find_report_name(result):
+    pattern = 'written to'
+    merged = 'merged_report'
+    for line in result:
+        idx = line.find(pattern)
+        idx1 = line.find(merged)
+        if idx > 0 and idx1 > 0:
+            return "_/" + line[idx + len(pattern):].replace(" ", "")
+
+    for line in result:
+        idx = line.find(pattern)
+        if idx > 0:
+            return "_/" + line[idx + len(pattern):].replace(" ", "")
+
+    return ''
+
+def create_foldor(name):
+    if not os.path.exists(name):
+        os.mkdir(name)
+        print("create successful")
+    else:
+        print("foldor has been created")
+
+def write_txt(foldor, text_name, text, version1, version2):
```

Comment:
which version stands for older version? version1 or version2?
try to use more meaningful name